lua/cinema/scoreboard/cl_playerlist.lua

```diff
@@ -51,11 +51,18 @@ concommand.Add("mute", function(ply, cmd, args, argss)
 
     if v then
         v.ClickMuted = not v.ClickMuted
-        print(v.ClickMuted and "Muted" or "Unmuted", v)
-        UpdateMutes()
+        UpdateMuteHistory(v)
+        print(v.ClickMuted and " Muted" or " Unmuted", v)
     end
 end)
 
+local history = util.JSONToTable(file.Read("swamp_mutehistory.txt", "DATA") or "") or {}
+function UpdateMuteHistory(ply)
+    history[ply:SteamID()] = (ply.ClickMuted or ply.IsChatMuted) and {mute=ply.ClickMuted, chatmute=ply.IsChatMuted} or nil
+    UpdateMutes()
```

Comment:
so you still check every single player when only updating one

Reply:
that's literally how it is in the current code except it runs every second